src/app/(main)/[lang]/page.tsx

```diff
@@ -82,34 +49,21 @@ const Home = async ({ params }: Props) => {
     );
   }
 
-  const initialLandingPage = await loadStudioQuery<PageBuilder>(
-    PAGE_QUERY,
-    { id: landingId, language: params.lang },
-    { perspective },
-  );
+  const initialData = {
+    ...initialLandingPage,
+    data: initialLandingPage.data,
+  };
 
-  if (!initialLandingPage.data) {
-    return (
-      <InformationSection
-        title="Landing Page is Missing Content"
-        body={`Your landing page is set, but it looks like there’s no content yet.\n Visit the Studio to start adding content and make your landing page come to life!`}
-        link={pagesLink}
-      />
-    );
-  }
```

Comment:
What if landingpage is not set, how will this look like for the end user?

Reply:
It will display the "Welcome! Velkommen! Välkommen!" information section that is returned further up here.